runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -368,6 +468,54 @@ else if (event.getGameState() == GameState.LOGIN_SCREEN)
 		}
 	}
 
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked menu)
+	{
+		int id = menu.getId();
```

Comment:
cant you get w/e you need from MenuOptionClicked event? You should not need to iterate menu entries

Reply:
I need a MenuEntry to get param0 and param1 which correspond to the x and y coordinates of LocalPoint representing the location of the object that was clicked. I don't see any way to get that directly from MenuOptionClicked. Am I missing something?

Reply:
`actionParam` is x and `widgetId` is y for clicks on game objects.

Reply:
It's not documented anywhere, I only knew it from looking at the Kourend Library plugin.